src/components/Bundle/Bundle.ts

```diff
@@ -82,6 +84,21 @@ function addListeners(bundle: Bundle) {
   bundle.addEventListener("input", bundle)
 }
 
+function initializeSelectedProducts(bundle: Bundle) {
+  const checkboxes = bundle.querySelectorAll<HTMLInputElement>('input[type="checkbox"][value]')
+  bundle.selectedProducts = []
+
+  checkboxes.forEach(checkbox => {
```

Comment:
can you convert this into more functional style?

bundle.selectedProducts = checkboxes.filter(...).map(...)